src/NuGet.Services.EndToEnd/Support/Clients/V2V3SearchClient.cs

```diff
@@ -15,15 +17,26 @@ namespace NuGet.Services.EndToEnd.Support
 {
     public class V2V3SearchClient
     {
+        /// <summary>
+        /// In case cloud service information is not provided, this will be the fallback instance count.
+        /// </summary>
+        private const int DefaultServiceInstanceCount = 2;
```

Comment:
@chenriksson - number of instances will never be below 2. This should be the config for auto scale, and also the depoyment config for search service in dev. If we have only a single instance we risk service unavailability.